tests/test_cache.py

```diff
@@ -463,8 +464,8 @@ def test_invalidate_new_object(self):
         eq_([a.val for a in u.addon_set.all()], [42, 17])
 
     def test_make_key_unicode(self):
-        translation.activate(u'en-US')
-        f = 'fragment\xe9\x9b\xbb\xe8\x85\xa6\xe7\x8e'
+        translation.activate('en-US')
+        f = 'fragment\xe9\x9b\xbb\xe8\x85\xa6\xe7\x8e\xa6'
```

Comment:
I think the previous value was significant. It's a valid sequence of bytes, which causes an error when used as an argument to `make_key` in Python2 (a bug that was fixed in this commit: https://github.com/django-cache-machine/django-cache-machine/commit/a767c52cbc3af9394eb931575e7049ec4e1fd275 )